src/index.html

```diff
@@ -11,6 +11,41 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="form"
```

Comment:
<img width="1301" alt="Screenshot 2023-07-07 at 11 12 23" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/53228259/37e4213b-2792-48fd-9005-ba1ef5bb9596">

Reply:
not fixed